### PR TITLE
feat: record bailout plugin

### PR DESCRIPTION
**Pull request by sxzz**
### Description

image

### CODE REVIEW

**Comment by antfu** [comment]
Good idea, maybe move next to "Transform Stack"?

**Comment by sxzz** [comment]
Done.

File: src/node/index.ts [hunk 1]
```diff
@@ -272,9 +273,9 @@ export default function PluginInspect(options: Options = {}): Plugin {
       const result = typeof _result === 'string' ? _result : _result?.code
       const sourcemaps = typeof _result === 'string' ? null : _result?.map
       const map = ssr ? transformMapSSR : transformMap
-      if (filter(id) && result != null) {
+      if (filter(id)) {
```
Comment (antfu) [inline src/node/index.ts:276]: I don't understand why we should record them? Returning `undefined` should indicate the plugin is bailed out.

Comment (sxzz) [reply src/node/index.ts:276]: I need to find out if a plugin has executed even if it bailed out. Also, figure out the plugin's order.

Comment (antfu) [reply src/node/index.ts:276]: Ok, I guess then for `result == null`, the label should better to be different from `no changes`, but maybe `bailout`?

Comment (sxzz) [reply src/node/index.ts:286]: Done

**Pull request merged**